vlsv_reader_parallel.cpp

```diff
@@ -164,7 +164,12 @@ namespace vlsv {
          parallelFileOpen = false;
       }
 
-      if (myRank == masterRank) filein.close();
+      if (myRank == masterRank)
+      { 
```

Comment:
Nitpicky code-formatting comment: The opening brace is in the wrong spot! (should be on the line above)

Reply:
Heh, for some definition of wrong!